src/charm.py

```diff
@@ -34,16 +39,23 @@ def __init__(self, *args):
                 ),
             ],
         )
+        self.metrics_endpoint_provider = MetricsEndpointProvider(
+            self,
+            relation_name="monitoring",
```

Comment:
At the cost of repeating myself: nooooooooes

Reply:
Agreed, fixed